src/components/ResetPassword/ResetPasswordModal/ResetPasswordModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import CommonModal from '../../../common/CommonModal/CommonModal';
+import ResetPasswordModalContent from './ResetPasswordModalContent';
+import styles from './ResetPasswordModal.module.css';
+import CheckMark from '../../../assets/BlueCheckMark.png';
+
+// TODO:
+// - Implement resetPassword
+
+const resetPassword = () => {
+  // Make request to reset password here
+  // eslint-disable-next-line no-console
+  console.log('password reset complete');
+};
+
+const ResetPasswordModal = ({ isOpen, onClose }) => {
+  const [modalStep, setModalStep] = useState(0);
+
+  const closeModal = () => {
+    setModalStep(0);
+    onClose();
+  };
+
+  return (
+    <CommonModal isOpen={isOpen} onClose={closeModal} modalClassName={styles.modal}>
```

Comment:
Justine updated the Figma so there is no longer a close button for this modal. The auth team added a prop to CommonModal so we can state if we do not want a close button ([in this PR](https://github.com/ctc-uci/find-your-anchor-frontend/pull/60)). Since that has not been merged into dev yet, can we add a TODO to remove the close button in the future?